### ISSUE TITLE
Log entries for deprecated code

### ISSUE THREAD

**Issue by SGXander** [bigbadblunt/homeassistant-lightwave2#124]
Hi, Hope this is still maintained as I'm getting startup errors for lightwave2 after a HA update:

Logger: homeassistant.helpers.frame
Source: helpers/frame.py:184
First occurred: 23:43:08 (18 occurrences)
Last logged: 23:43:47

    Detected that custom integration 'lightwave2' calls async_forward_entry_setup for integration, lightwave2 with title: ****@**** and entry_id: ****, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/lightwave2/__init__.py, line 131: hass.async_create_task(forward_setup(config_entry, "cover")), please report it to the author of the 'lightwave2' custom integration
    Detected that custom integration 'lightwave2' calls async_forward_entry_setup for integration, lightwave2 with title: ****@**** and entry_id: ****, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/lightwave2/__init__.py, line 132: hass.async_create_task(forward_setup(config_entry, "binary_sensor")), please report it to the author of the 'lightwave2' custom integration
    Detected that custom integration 'lightwave2' calls async_forward_entry_setup for integration, lightwave2 with title: ****@**** and entry_id: ****, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/lightwave2/__init__.py, line 133: hass.async_create_task(forward_setup(config_entry, "sensor")), please report it to the author of the 'lightwave2' custom integration
    Detected that custom integration 'lightwave2' calls async_forward_entry_setup for integration, lightwave2 with title: ****@**** and entry_id: ****, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/lightwave2/__init__.py, line 134: hass.async_create_task(forward_setup(config_entry, "lock")), please report it to the author of the 'lightwave2' custom integration
    Detected that custom integration 'hacs' accesses hass.helpers.event. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from event directly at custom_components/hacs/base.py, line 630: self.hass.helpers.event.async_track_time_interval(, please create a bug report at https://github.com/hacs/integration/issues

**Comment by hargcore** [comment]
Spotted the same thing - lots of deprecated items in the log.

**Comment by xela1** [comment]
I don't believe this is being maintained any more, there is a fork maintained by an employee of lightwave themselves (@ikb42) that is actively maintained and looking at his current branch has some good work in progress changes going in.

https://github.com/LightwaveSmartHome/homeassistant-lightwave-smart

It's been a while since I switched, but if I recall correctly it does change entity ids so you'll need to find a good time to switch

I would like to thank @bigbadblunt again for all his hard work originally on this.

**Comment by bigbadblunt** [comment]
That's right. I don't have any lightwave devices any more, so can't really update this code any more without anything to test against. I recommend moving to the fork linked above. 

@xela1 thanks for the support.
